### ISSUE TITLE
Does not work!

### ISSUE THREAD

**Issue by envatic** [voracious/vite-plugin-node-polyfills#1]
failed to load config from /Users/steve/sites/dfxswap.io/vite.config.js
error when starting dev server:
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /Users/steve/sites/dfxswap.io/node_modules/vite-plugin-node-polyfills/package.json
    at new NodeError (node:internal/errors:387:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:439:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:663:7)
    at resolveExports (node:internal/modules/cjs/loader:493:36)
    at Module._findPath (node:internal/modules/cjs/loader:533:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:942:27)
    at Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1022:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/steve/sites/dfxswap.io/vite.config.js:83:41)

**Comment by voracious** [comment]
Hey @envatic. Thanks for sharing the stack trace. Any chance you could provide a reproducible example or a link to your project?

**Comment by fxsalazar** [comment]
Same for me:
```
error when starting dev server:
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in pp/node_modules/vite-plugin-node-polyfills/package.json
    at new NodeError (node:internal/errors:387:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:464:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:688:7)
    at resolveExports (node:internal/modules/cjs/loader:493:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:533:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:942:27)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (**/vite.config.js:35:49)
```

**Comment by voracious** [comment]
This appears to be due to the package exporting a `main` field that is not in the CommonJS format. I'll remove that field for now (in hopes that your project correctly imports the ESM format), and I'll add it back once I get a chance to thoroughly test a CJS example.

**Comment by voracious** [comment]
I just released `v0.2.0` with the `main` field removed. Any chance one of you can try out the new version to see if it works for you?

**Comment by voracious** [comment]
The latest version `v0.3.2` works with CommonJS now!
